### ISSUE TITLE
Dear sir,      Can you give me the hyperparameter setting for DivideMix on this dataset?

### ISSUE THREAD

**Issue by LanXiaoPang613** [weijiaheng/Advances-in-Label-Noise-Learning#2]
Dear sir, 
    Can you give me the hyperparameter setting for DivideMix on the CIFAR10N&CIFAR100N? When i reproduce the experiments of dmix on cifar10N, the results for aggregate based on ResNet-34 and default setting of divideMix are only nearly 84%. So if possible, pls give me the hyper-parameter setting for the divideMix.
